### PR TITLE
feat: Create group chats

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: ui/app/AppLayouts/Chat/components/GroupChatPopup.qml [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    Input {
+        id: searchBox
+        visible: selectChatMembers
+        placeholderText: qsTr("Search")
+        icon: "../../../img/search.svg"
+        iconWidth: 17
+        iconHeight: 17
+        onKeyReleased: function(){
+            
+        }
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/components/GroupChatPopup.qml:87]: Leftover?

File: ui/app/AppLayouts/Chat/components/GroupChatPopup.qml [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+          anchors.top: lblNewGroup.bottom
+          text: qsTr("%1 / 10 members").arg(memberCount)
+          color: Theme.darkGrey
+          font.family: "Inter"
+          font.pixelSize: 15
+      }
+    }
+
+    Input {
+        id: searchBox
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/components/GroupChatPopup.qml:79]: Can you use the SearchBox component?

File: ui/shared/Input.qml [hunk 3]
```diff
@@ -70,6 +71,7 @@ Item {
             background: Rectangle {
                 color: "#00000000"
             }
+            Keys.onReleased: onKeyReleased()
```
Comment (jrainville) [inline ui/shared/Input.qml:74]: You probably can access that using `textField.Keys.onReleased`. I don't think we need to add this new prop

File: ui/app/AppLayouts/Chat/components/GroupChatPopup.qml (hunk 1 above)
Comment (richard-ramos) [reply ui/app/AppLayouts/Chat/components/GroupChatPopup.qml:87]: oops!

File: ui/shared/Input.qml (hunk 3 above)
Comment (richard-ramos) [reply ui/shared/Input.qml:74]: cool! I'll try that. I did not know that was possible!

**Pull request merged**